release-notes.md

```diff
@@ -1,5 +1,11 @@
 # Service Broker API Release Notes #
 
+## <a id='2-12'></a>v2.12 ##
+2017-06-13
+
+* Added `context` field to request body for provisioning a service instance (`PUT /v2/service_instances/:instance_id`) and updating a service instance (`PATCH /v2/service_instances/:instance_id`). Also added a `profile.md` file describing recommended usage paterns for environment-specific extensions and variations.
+* The specification now uses RFC2119 keywords to make it clearer when things are required rather than optional or recommended.
```

Comment:
I wasn't intending it to be complete, but I think it should contain all changes that aren't basic formatting / typos. Updating now.